CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Bug fixes
 
 * [#PR363](https://github.com/juxt/crux/pull/363) Allow `full-results?` and other boolean flags in a vector-style query
+* [#362](https://github.com/juxt/crux/issues/362) Fixes 362 where hashes of small maps were dependent on oder *Breaking*
```

Comment:
thai-po ;)

Reply:
I'd consider this part of the breaking changes section, above (might even be worth explicitly calling this 'breaking changes'?

Reply:
Should we do this for all previous versions as well then? I was following the style of the log previously. I think it would make it stand out more, but since we only have a few items per version anyway I don't think it needs to be that displayed. Although I'm fine to be overruled here if you disagree

Reply:
let's stick with the convention for now and discuss whether to change it - could you move mine in while you're there?

Reply:
will do